Provide example showing how to modify log level (#723)
redpanda-data/documentation: docs/reference/rpk/rpk-redpanda/rpk-redpanda-admin-config-log-level.mdx

    </tr>
  </tbody>
</table>

### Example

> Let's say you have an issue and you want to see more details in the log. Open a terminal on a Redpanda node and run the following command:

Reviewer: ```suggestion
If you want to see more details in the log, run the following command:
```

Since this is the CLI reference page, it's safe to assume that the user is already with an open terminal in a RP node

```bash
journalctl -u redpanda
```

This will print out the log since the node's last start. You can add `-f` to the command above to tail logs. Most lines follow this pattern:

```
<log date> <hostname> <command>: <loglevel> <event date> [shard <shard index>] <subsystem> - <message>
```

Pay attention to the `subsystem` in each line. You can modify the log level for each subsystem with a command similar to the following:

```bash
rpk redpanda admin config log-level set cloud_storage -l trace --host <hostname_or_ip>:9644
```

> The above command enables `trace` level logging for the `cloud_storage` subsystem. The valid logging level options are `error`, `warn`, `info`, `debug`, and `trace` (the default level is `debug`). There are a number of subsystems, and enabling a more detailed level of logging is helpful for some moreso than others.

Reviewer 2: Suggestion:

Each subsystem can have its own log level. Valid logging level options are `error`, `warn`, `info`, `debug`, and `trace`  To get the most detailed level of logging for the `cloud_storage` subsystem, for example, set the log level to `trace` as shown:

```bash
rpk redpanda admin config log-level set cloud_storage -l trace --host <hostname_or_ip>:9644
```
 To view the last ten lines of the log, include the `-f` flag in the command above.
